src/components/ProductDetail.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import '../index.css';
+
+const ProductDetail = (props) => {
```

Comment:
This is okay, but what if you wanted to show one single product detail somewhere in your app.